Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedListItemView.swift

```diff
@@ -77,7 +70,7 @@ public struct PreplannedListItemView: View {
                 Image(systemName: "arrow.down.circle")
             }
             .buttonStyle(.plain)
-            .disabled(!model.canDownload)
+            .disabled(!model.status.allowsDownload)
```

Comment:
moving canDownload (allowsDownload) to the status allows us to test the status individually and not test the model each time the status of the model changes.